bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/P110.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tapocontrol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please update the default translation properties in i18n folder to be in sync with all your changes

Reply:
updated default translation properties in i18n
`thing-type.tapocontrol.L510.label = L510 Series White-Bulb
thing-type.tapocontrol.L510.description = Tapo Smart dimmable White-Light-Bulb
thing-type.tapocontrol.L530.label = L530 Series Color-Bulb
thing-type.tapocontrol.L530.description = Tapo Smart Multicolor Light-Bulb`

Reply:
You did not add entries for your new thing types.

Ideally, you should just run the command: `mvn i18n:generate-default-translations`

Reply:
Did `mvn clean install`. I never had to do this or edit this file manually before....

When running ` mvn i18n:generate-default-translations` i got an warning and error: `[WARNING] The POM for org.openhab.core.tools:i18n-maven-plugin:jar:3.3.0-SNAPSHOT is missing`
If i change the openhab-addons\pom.xml from `${ohc.version}` to `3.2.0` generating translations works fine.
```<plugin>
          <groupId>org.openhab.core.tools</groupId>
          <artifactId>i18n-maven-plugin</artifactId>
          <version>${ohc.version}</version>    <----- change this to 3.2.0
    </plugin>
```
changed it back and did clean install again. now my translation properties are fixed